code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -89,6 +91,47 @@
 	..()
 	return TRUE
 
+/datum/reagent/medicine/C2/probital
+	name = "Probital"
+	description = "Originally developed as a prototype-gym supliment for those looking for quick workout turnover, this oral medication quickly repairs broken muscle tissue but causes lactic acid buildup, tiring the patient. Overdosing can cause extreme drowsiness. An Influx of nutrients promotes the muscle repair even further."
+	reagent_state = SOLID
+	color = "#FFFF6B"
+	overdose_threshold = 20
+
+/datum/reagent/medicine/C2/probital/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-2.25*REM, FALSE)
+	var/ooo_youaregettingsleepy = 3.5
+	switch(round(M.getStaminaLoss()))
+		if(10 to 40)
+			ooo_youaregettingsleepy = 3
+		if(41 to 60)
+			ooo_youaregettingsleepy = 2.5
+		if(61 to 200) //you really can only go to 120
+			ooo_youaregettingsleepy = 2
+	M.adjustStaminaLoss(ooo_youaregettingsleepy*REM, FALSE)
+	..()
+	. = TRUE
+
+/datum/reagent/medicine/C2/probital/overdose_process(mob/living/M)
+	M.adjustStaminaLoss(3*REM, 0)
+	if(M.getStaminaLoss() >= 80)
+		M.drowsyness++
+	if(M.getStaminaLoss() >= 100)
+		to_chat(M,"<span class='warning'>You feel more tired than you usually do, perhaps if you rest your eyes for a bit...</span>")
+		M.adjustStaminaLoss(-100, TRUE)
```

Comment:
I can't wait for true gamers to intentionally OD themselves on both probital and modafinil, then brush off disabler shots like they're nothing (does the stun from staminacrit get cleared instantly when you leave it, or do you still remain stunned for a while?).

The slowdown from the stamina damage will still be an issue, of course, especially since ignore_slowdown() STILL DOESN'T WORK, but I'm sure someone will find a way around that.

Reply:
modafinil has it's own issues, if you want to get popped with lethals on top of that go ahead 😏

Reply:
@ATH1909 Morphine